src/test/ui/parser/several-carriage-returns-in-doc-comment.rs

```diff
@@ -0,0 +1,8 @@
+// Issue #62863
+// ignore-tidy-cr
+
+/// This do
c comment contains 
three isolated `\r`
 symbols
```

Comment:
Maybe mention in a comment that this might be rendered as `^M`? It looks kinda funny on GitHub.

Reply:
good call, fixed!